rest/src/main/java/fi/otavanopisto/pyramus/rest/CourseRESTService.java

```diff
@@ -579,11 +580,15 @@ public Response createCourseStudent(@PathParam("COURSEID") Long courseId, fi.ota
     BigDecimal lodgingFee = null;
     Currency lodgingFeeCurrency = null;
 
-    return Response.status(Status.OK).entity(
-        objectFactory.createModel(courseController.createCourseStudent(course, student, enrolmentType, participantionType,  
-            toDate(entity.getEnrolmentTime()), entity.getLodging(), optionality, billingDetails, lodgingFee, lodgingFeeCurrency, 
-            organization, additionalInfo, room)))
-      .build();
+    try {
```

Comment:
Check whether course already contains this student. If it does return bad request